internal/command/commands/env.go

```diff
@@ -57,6 +59,9 @@ func (e *Env) Run(args []string) int {
 	}
 
 	if e.Status {
+		if *name == "" {
+			*name = utils.FetchKey("EnvName")
```

Comment:
can we create a constant for the key "EnvName" and use the constant everywhere? this way if the key changes then you would need to just change the constant value.